src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParameters.cs

```diff
@@ -241,5 +246,7 @@ private static void AddParameters(IMethodSymbol method, ConcurrentDictionary<IPa
 
             return true;
         }
+
+        protected abstract bool IsInlineRecord(IMethodSymbol methodSymbol);
```

Comment:
The official name for this kind of records is "Positional Record", and for the constructor that it gets generated by the compiler is called "Primary Constructor". So, I'd suggest renaming the method to "IsPrimaryConstructor" or something like that.

Reply:
I was looking for the name but couldn't find it... Could you point me to the source please?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/records#positional-record-members